<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyIDE</class>
 <widget class="QMainWindow" name="PyIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyIDE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 36, 40);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="commandEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 41, 46);
color: white;
border: 1px solid black;</string>
    </property>
    <property name="tabStopWidth">
     <number>32</number>
    </property>
    <property name="tabStopDistance">
     <double>32.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(171, 171, 171);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultText">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>381</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 41, 46);
color: rgb(0, 170, 0);
border: 1px solid black;</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: none;
</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="shortcut">
     <string>F5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: none;
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="shortcut">
     <string>F6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: none;
</string>
    </property>
    <property name="text">
     <string>Save (.py)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
